domain/schema/model/sql/0022-resource.sql

```diff
@@ -151,3 +151,32 @@ CREATE TABLE resource_image_store (
     FOREIGN KEY (store_uuid)
     REFERENCES resource_container_image_metadata_store (uuid)
 );
+
+
+CREATE VIEW v_application_resource AS
+SELECT
+    r.uuid,
+    ar.application_uuid,
+    r.charm_resource_name AS name,
+    r.last_polled
+FROM resource AS r
+INNER JOIN application_resource AS ar ON r.uuid = ar.resource_uuid;
+
+CREATE VIEW v_resource AS
+SELECT
+    r.uuid,
+    ar.application_uuid,
+    r.charm_resource_name AS name,
+    r.created_at,
+    r.revision,
+    r.origin_type_id,
+    rrb.name AS retrieved_by,
+    rrbt.name AS retrieved_by_type,
+    cr.path,
+    cr.description,
+    cr.kind_id
+FROM resource AS r
+INNER JOIN application_resource AS ar ON r.uuid = ar.resource_uuid
```

Comment:
`INNER` is implicit. Just use `JOIN`.

Reply:
I agree, but linter don't: 

https://github.com/juju/juju/actions/runs/12035598746/job/33554940028 :(